### ISSUE TITLE
Please push the updates from the developments at POC21

### ISSUE THREAD

**Issue by shulter** [osefrance/SolarConcentrator#6]
sources seem outdated right now.

**Comment by Cywil23** [comment]
Yes, the last version of the 3D model is here : http://a360.co/1Vz5g9C
When design will be finalized, we will model again the demonstrator with FreeCAD to publish source files done by an open source software
